libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -64,6 +64,13 @@ describe('Schema Merge Tests', function () {
             // Try installing latest bf if the schema changed to make sure the
             // discrepancy is not because we are using a different version of the CLI
             // and we ensure it is installed while on it.
+
+            // When installing bf-cli, there is sometimes a prompt during install to allow telemetry.
+            // We need to set an environment variable so the prompt doesn't appear and halt install, causing a timeout.
+            const telemSettingPath = 'BF_CLI_TELEMETRY';
+            const originalTelemSetting = process.env[telemSettingPath];
+            process.env[telemSettingPath] = false;
```

Comment:
There's actually a way to [pass a custom set of environment variables](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) when calling `exec`. In my opinion, it's a little cleaner as you aren't trouncing any global state. It also makes clear where this side effect is necessary.

Reply:
Ah, I forgot about that! Making the adjustment, now.